app/assets/javascripts/sufia/save_work/save_work_control.es6

```diff
@@ -29,6 +29,19 @@ export class SaveWorkControl {
   }
 
   /**
+   * Keep the form from submitting (if the return key is pressed)
+   * unless the form is valid.
+   *
+   * This seems to occur when focus is on one of the visibility buttons
+   */
+  preventSubmitUnlessValid() {
+    this.form.on('submit', (evt) => {
```

Comment:
It is using html5 form validations on certain fields, but not the hidden ones (associated files).